src/css/embed.scss

```diff
@@ -72,3 +72,17 @@ ul {
 .u-hidden {
     display: none;
 }
+
+.u-underline {
```

Comment:
Should be fine, @philwills, the interactive lives in its own iframe, so it won't interfere with styles defined in the frontend